libs/external-db-postgres/src/postgres_data_provider.ts

```diff
@@ -53,12 +53,16 @@ export default class DataProvider implements IDataProvider {
                                                           .catch( translateErrorCodes )
                                 return rs.rowCount
                         } ) )
-        return res.reduce((sum, i) => i + sum, 0)
+        
+        // @ts-ignore
+        return res.reduce((sum: number, i: number) => i + sum, 0)
     }
 
-    async delete(collectionName: string, itemIds: string[]) {
+    async delete(collectionName: string, itemIds: string[]): Promise<number> {
         const rs = await this.pool.query(`DELETE FROM ${escapeIdentifier(collectionName)} WHERE _id IN (${prepareStatementVariables(itemIds.length)})`, itemIds)
                              .catch( translateErrorCodes )
+        
+        // @ts-ignore
         return rs.rowCount
     }
```

Comment:
@Idokah  
I have no idea why, but the CI didn't want to compile it and claimed that update / delete can return an answer that is null ( you can see the errors near the failed comments)
No matter what I did, it wouldn't compile the Postgres code. 
That's why I added `@ts-ignore`